### DISCUSSION TITLE
select(.data, kolom yg ingin diambil untuk dplyr

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#547] [Q&A] [answered]
bagian error pada bagian perintah code ini, apakah ada yang salad dalam code ini ya?
<img width="1440" alt="Screen Shot 2023-02-18 at 20 20 31" src="https://user-images.githubusercontent.com/123171973/219868256-9cb1fd73-bc24-473d-9738-69fd64b4ec35.png">

**Comment by VictorNugraha** [comment, marked as answer]
Dear @mosesnugraha,

Bisa dicoba untuk menghilangkan tanda **_""_** ketika memilih kolom _Paperlessdate_.

Best Regards,

Team Algoritma
